src/main/kotlin/no/nav/familie/ba/sak/kjerne/eøs/kompetanse/KompetanseController.kt

```diff
@@ -81,5 +81,32 @@ class KompetanseController(
         if (oppdatertKompetanse.barnAktører.isEmpty()) {
             throw FunksjonellFeil("Mangler barn", httpStatus = HttpStatus.BAD_REQUEST)
         }
+
+        if (
+            (oppdatertKompetanse.erAnnenForelderOmfattetAvNorskLovgivning == true && oppdatertKompetanse.søkersAktivitet?.gyldigForAnnenForelder == false) ||
+            (oppdatertKompetanse.erAnnenForelderOmfattetAvNorskLovgivning == false && oppdatertKompetanse.søkersAktivitet?.gyldigForSøker == false)
+        ) {
+            "Valgt verdi for søkers aktivitet er ikke gyldig ${if (oppdatertKompetanse.erAnnenForelderOmfattetAvNorskLovgivning) "når annen forelder er omfattet av norsk lovgivning" else ""}"
+                .trim()
+                .apply {
+                    throw FunksjonellFeil(
+                        this,
+                        this,
+                    )
+                }
```

Comment:
Trenger vi å trimme dette? Dersom vi ikke setter frontendFeilmelding vil den bli lik som meldingen. 
```suggestion
            throw FunksjonellFeil(
                        "Valgt verdi for søkers aktivitet er ikke gyldig ${if (oppdatertKompetanse.erAnnenForelderOmfattetAvNorskLovgivning) "når annen forelder er omfattet av norsk lovgivning" else ""}"
                    )
                }
```

Reply:
Ah, antok at det var likt som for Feil, at frontendFeilmelding blir null om den ikke er satt. Kan skrive det om.

Trim er fordi vi har et space etter "ikke gyldig", så hvis `erAnnenForelder...` er false vil teksten avslutte med et space

Reply:
[6093175](https://github.com/navikt/familie-ba-sak/pull/3766/commits/6093175bcf71b18dad575a1fd24fecd551e9168f)